sdk-transaction/src/main/java/org/fisco/bcos/sdk/transaction/codec/decode/TransactionDecoderInterface.java

```diff
@@ -57,7 +56,7 @@
      */
     public TransactionResponse decodeReceiptWithValues(
             String abi, String functionName, TransactionReceipt receipt)
-            throws JsonProcessingException, TransactionException, IOException, ABICodecException;
+            throws TransactionException, IOException, ABICodecException;
```

Comment:
Don't recommend to update the interface since it's a old public interface.

Reply:
JsonProcessingException is sub-class of IOException.